### PR TITLE
[Android] Update Android warnings to ignore for now

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

Update warnings to ignore so we can build without issues on Beta channel 15.4 preview 5.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Platform.Android.FormsViewGroup/Xamarin.Forms.Platform.Android.FormsViewGroup.csproj [hunk 1]
```diff
@@ -27,6 +27,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NoWarn>CS0109</NoWarn>
```
Comment (hartez) [inline Xamarin.Forms.Platform.Android.FormsViewGroup/Xamarin.Forms.Platform.Android.FormsViewGroup.csproj]: Pretty sure we only need this in the Android platform project.

Comment (rmarinho) [inline Xamarin.Forms.Platform.Android.FormsViewGroup/Xamarin.Forms.Platform.Android.FormsViewGroup.csproj]: Did you git clean -xfd and build the all sln ? just on the plattform didn't work for me.

Comment (hartez) [inline Xamarin.Forms.Platform.Android.FormsViewGroup/Xamarin.Forms.Platform.Android.FormsViewGroup.csproj]: I may not have. No worries, should only be temporary anyway.

**Comment by rmarinho** [comment]
build

**Comment by hartez** [comment]
Remove these once https://bugzilla.xamarin.com/show_bug.cgi?id=58533 is fixed.

**Pull request merged**